examples/language-modeling/run_mlm_flax.py

```diff
@@ -566,8 +567,9 @@ def tokenize_function(examples):
     ).create(model.params)
 
     # Create learning rate scheduler
+    # warmup_steps = 0 causes the Flax optimizer to return NaNs; warmup_steps = 1 is functionally equivalent.
```

Comment:
We could a do a `logger.warning(...)` if `training_args.warmup_steps` happens to be 0.

Reply:
Just added !